docs/source/models.rst

```diff
@@ -615,6 +615,37 @@ scored with both DataFrame input and numpy array input. Finally, you can use the
 function in Python or `mlflow_load_model <R-api.rst#mlflow-load-model>`__ function in R to load MLflow Models
 with the ``keras`` flavor as `Keras Model objects <https://keras.io/models/about-keras-models/>`_.
 
+Keras pyfunc usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a minimal Sequential model, an example configuration for the pyfunc predict() method is:
+
+.. code-block:: py
+    
+    import mlflow
+    import numpy as np
+    from tensorflow import keras
+
+    with mlflow.start_run():
+        X = np.array([-2, -1, 0, 1, 2, 1]).reshape(-1, 1)
+        y = np.array([0, 0, 1, 1, 1, 0])
+        model = keras.Sequential(
+            [
+                keras.Input(shape=(1,)),
+                keras.layers.Dense(1, activation="sigmoid"),
+            ]
+        )
+        model.compile(loss="binary_crossentropy", optimizer="adam", metrics=["accuracy"])
+        model.fit(X, y, batch_size=3, epochs=5, validation_split=0.2)
+
+        model_info = mlflow.keras.log_model(keras_model=model, artifact_path="model")
+
+    keras_pyfunc = mlflow.pyfunc.load_model(model_uri=model_info.model_uri)
```

Comment:
In order to demonstrate the ability to write the model artifact to an explicit local directory location, perhaps showcase the arg that permits this?
i.e.,

```python

# line 625
import os
import shutil

local_artifact_dir = "/tmp/mlflow/keras_model"
os.makedirs(local_artifact_dir)
keras_pyfunc = mlflow.pyfunc.load_model(model_uri=model_info.model_uri, dst_path=local_artifact_dir)

# line 648
shutil.rmtree(local_artifact_dir)

Reply:
Thanks for the pointer @BenWilson2! I've added the use of the `dst_path` arg. Used `pathlib` to create the folder instead of `os`

Reply:
good idea :)